web/src/components/Wrapper.js

```diff
@@ -1,41 +1,43 @@
 import PropTypes from 'prop-types';
-import React, { Component } from 'react';
+import React, { useEffect } from 'react';
+import { withRouter } from 'react-router';
 import Header from './Header';
 import Footer from './Footer';
 
-class Wrapper extends Component {
-  componentDidMount() {
-    const { isDev } = this.props;
+const grayPaths = ['/create-account', '/edit-account', '/login', '/me'];
+
+const Wrapper = ({ children, isDev, location: { pathname } }) => {
+  useEffect(() => {
     if (isDev) {
-      this.addTota11y();
+      const script = document.createElement('script');
+      script.src = '/_dev/tota11y.min.js';
```

Comment:
We probably need an issue to remove this, since I don't think we've actually used it in recent memory.

Reply:
Agreed